<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerXClass</class>
 <widget class="QMainWindow" name="VideoPlayerXClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPlayerX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoControlWidget" name="videoWidget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #000;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSlider" name="positionSlider">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="volumeBtn">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/喇叭.png</normaloff>:/VideoPlayerX/res/喇叭.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="playBtn">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/播放.png</normaloff>:/VideoPlayerX/res/播放.png</iconset>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseBtn">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/暂停.png</normaloff>:/VideoPlayerX/res/暂停.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/停止.png</normaloff>:/VideoPlayerX/res/停止.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="leftBtn">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/快退.png</normaloff>:/VideoPlayerX/res/快退.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rightBtn">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/快进.png</normaloff>:/VideoPlayerX/res/快进.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openBtn">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/弹出.png</normaloff>:/VideoPlayerX/res/弹出.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="positionLab">
         <property name="text">
          <string>00:00:00/00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="optionButton">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/菜单.png</normaloff>:/VideoPlayerX/res/菜单.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fullscreenBtn">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="VideoPlayerX.qrc">
           <normaloff>:/VideoPlayerX/res/全屏.png</normaloff>:/VideoPlayerX/res/全屏.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoControlWidget</class>
   <extends>QWidget</extends>
   <header>VideoControlWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="VideoPlayerX.qrc"/>
 </resources>
 <connections/>
</ui>
